doc/using.but

```diff
@@ -980,8 +980,8 @@ follows:
 \cq{o} for odd, \cq{e} for even, \cq{m} for mark and \cq{s} for space.
 
 \b A single upper-case letter specifies the flow control: \cq{N} for
-none, \cq{X} for XON/XOFF, \cq{R} for RTS/CTS and \cq{D} for
-DSR/DTR.
+none, \cq{X} for XON/XOFF, \cq{R} for RTS/CTS, \cq{D} for
```

Comment:
In `RTS/CTS` the slash is intended to be an OR like in some technical or non-technical contexts.

Or is it intended as RTS and CTS.

---

Just a non-trivial nit-pick for the docs.

Reply:
To avoid stupid questions from anyone, I have read the commit message.

thanks